### PR TITLE
PP-838: add support for USDRIF

### PR DESCRIPTION
**Pull request by antomor**
## What

- Add an exchange that returns the USDRIF price

## Why

- USDRIF isn't supported

### CODE REVIEW

**Comment by antomor** [comment]
Once it's merged, we need to release a new version and update the server accordingly

File: src/api/pricer/BaseExchangeApi.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import type { BigNumber as BigNumberJs } from 'bignumber.js';
 
-export type BaseCurrency = 'TRIF' | 'RIF' | 'RDOC' | 'RBTC' | 'TKN';
+export type BaseCurrency = 'TRIF' | 'RIF' | 'RDOC' | 'RBTC' | 'TKN' | 'USDRIF';
```
Comment (franciscotobar) [inline src/api/pricer/BaseExchangeApi.ts:3]: What is going to be the name in testnet, `USDRIF` aswell?

**Review by franciscotobar — APPROVED** [review]
LGTM

File: src/api/pricer/BaseExchangeApi.ts (hunk 1 above)
Comment (antomor) [reply src/api/pricer/BaseExchangeApi.ts:3]: Yes, it looks like that indeed. I've checked the name of the [testnet contract](https://explorer.testnet.rsk.co/address/0x8dbf326e12a9ff37ed6ddf75ada548c2640a6482).

**Pull request merged**